public/js/graph/react_graph.js

```diff
@@ -331,11 +364,28 @@ var Graph = React.createClass({
                     edgesJSON={this.state.edgesJSON}
                     svg={this}/>
                 <EdgeGroup svg={this} ref='edges' edgesJSON={this.state.edgesJSON}/>
-            </svg>
+            </svg></div>
+
         );
     }
 });
 
+var Button = React.createClass({
+
+    handleMouseDown: function() {
```

Comment:
You'll need to change the `mouseDown` prop before removing the function (see above comment).